lightly/utils/dependency.py

```diff
@@ -3,24 +3,38 @@
 
 @functools.lru_cache(maxsize=1)
 def torchvision_vit_available() -> bool:
+    """Checks if Vision Transformer (ViT) models are available in torchvision.
+
+    This function checks if the `vision_transformer` module is available in torchvision,
+    which requires torchvision version >= 0.12. It also handles exceptions related to 
+    CUDA version mismatches and installation issues.
+
+    Returns:
+        bool: True if the Vision Transformer (ViT) models are available in torchvision, 
```

Comment:
The type is already in the typing of the function, no need to tell it twice.
```suggestion
       True if the Vision Transformer (ViT) models are available in torchvision, 
```